lib/filters/global/actions.js

```diff
@@ -17,7 +17,12 @@ module.exports = {
                     body.format = 'json';
                     return restbase[req.method](req)
                     .then(function(res) {
-                        if (res.status !== 200) {
+                        if (
+                            res.status !== 200
```

Comment:
Hmm, maybe a 500 is better.  We don't necessarily know that the request was poorly formatted -- all we know is that the PHP action API didn't give us back what we expect.